seed/energy/meter_data_processor/green_button_parser.py

```diff
@@ -0,0 +1,170 @@
+import logging
+
+from seed.energy.meter_data_processor import green_button_maps
+_log = logging.getLogger(__name__)
+
+root_ns = '{http://www.w3.org/2005/Atom}'
+gb_ns = '{http://naesb.org/espi}'
+
+def get_link_href_head(link):
+    href = link.get('href')
+    if href[0:1]=='/':
+        href = href[1:]
+    first_slash = href.find('/')
+    return href[0:first_slash]
+
+def get_link_href_tail(link):
+    href = link.get('href')
+    last_slash = href.rfind('/')
+    return href[(last_slash+1):]
```

Comment:
You might consider using the [`urlparse`](https://docs.python.org/2/library/urlparse.html) library for this functionality.